package.json

```diff
@@ -47,7 +47,6 @@
     "@babel/types": "7.19.3",
     "esbuild": "0.19.7",
     "jest-snapshot": "29.3.1",
-    "msw": "patch:msw@npm:0.40.2#.yarn/patches/msw-npm-0.40.2-2107d48752",
```

Comment:
can you also delete `.yarn/patches/msw-npm-0.40.2-2107d48752`?

Reply:
Yes sirrrrr